docs/maatregelen/maak_de_vereiste_onderdeel_van_programma_van_eisen.md

```diff
@@ -0,0 +1,108 @@
+- documentatie_beoordeling_niet_hoog_risico_ai
+- persoonsgegevens_worden_rechtmatig_verwerkt
+- fundamentele_rechten
+- geb_dpia_verplicht_bij_hoog_risico
+- juistheid_en_actualiteit_van_persoonsgegevens
+- kwaliteitsbeheersysteem_voor_hoog_risico_ai
+- kwaliteitscriteria_voor_data
+- maatregelen_van_gebruiksverantwoordelijken_voor_gebruik
+- beoordelen_gevolgen_voor_grondrechten
+- monitoring_na_het_in_de_handel_brengen
+- non_discriminatie
+- ontwerp_voor_nauwkeurigheid_robuustheid_en_cyberbeveiliging
+- minimale_verwerking_van_persoonsgegevens
+- privacy_bij_ontwerp_bij_verwerking_van_persoonsgegevens
+- inroepen_privacyrecht_bij_verwerking_persoonsgegevens
+- recht_op_niet_geautomatiseerd_besluitvorming
+- recht_op_uitleg_ai_besluiten
+- risicobeheersing
+- risicobeoordeling_voor_jongeren_en_kwetsbaren
+- technische_documentatie_voor_hoog_risico_ai
+- toezichtmogelijkheden_voor_gebruikers
+- traceerbare_besluitvorming
+- transparantie
+- transparantie_bij_verwerken_persoonsgegevens
+- verplicht_risicobeheersysteem_voor_hoog_risico_ai
+- Verplichtingen_van_aanbieders_van_ai_modellen_voor_algemene_doeleinden_
+- Verplichtingen_van_aanbieders_van_ai_modellen_voor_algemene_doeleinden_met_systeemrisico
+- ai_modellen_algemene_doeleinden_syteemrisico_informatie_ernstige_incidenten
+- ai_modellen_algemene_doeleinden_syteemrisico_cyberbeveiliging
+- registratieverplichtingen_hoog_risico
+- conformiteitsbeoordelingsprocedure_hoog_risico
+- aantoonbaarheid_vereisten_hoog_risico
+- toegankelijkheidseisen_hoog_risico
+- verstrekking_van_informatie_op_verzoek
+- wettelijke_verwerking_van_gevoelige_gegevens
+- recht_op_toegang_tot_publieke_informatie
+- algoritmeregister
+- beginsel_van_proportionaliteit_en_subsidiariteit
+
+# vul hier de fasen van de levenscyclus in die horen bij deze maatregel
+levenscyclus: 
+- ontwerp
+- ontwikkelen
+# vul hier de bouwblokken in die horen bij deze maatregel
+bouwblok: 
+- publieke-inkoop
+# geef hier aan welke rollen bij deze maatregelen betrokken kunnen zijn
+rollen:
+- proceseigenaar
+- behoeftesteller
+- data scientist
+- inkoopadviseur
+- contractbeheerder
+- aanbestedingsjurist
+
+---
+
+<!-- Let op! onderstaande regel met 'tags' niet weghalen! Deze maakt automatisch de knopjes op basis van de metadata  -->
+<!-- tags -->
+
+## Maatregel
+<!-- Vul hier een omschrijving in van wat deze maatregel inhoudt. -->
+ Maak de vereiste onderdeel van het programma van eisen
+
+## Toelichting
+<!-- Geef hier een toelichting van deze maatregel -->
+Door de vereiste onderdeel te maken van het programma van eisen, is het voor aanbieders duidelijk dat hun oplossing hieraan moet voldoen. Afhankelijk van de specifieke toepassing, context en noodzaak kan een vereiste in het programma van eisen concreet worden gemaakt. Het is hierbij van belang om dit af te wegen tegen zaken die mogelijk al zijn geregeld door middel van algemene inkoopvoorwaarden die gelden voor algoritmes en AI.   
```

Comment:
Wat zijn de 'algemene inkoopvoorwaarden die gelden voor algoritmes en AI'? Kunnen we een verwijzing toevoegen?